Mathlib/Data/Int/Bitwise.lean

```diff
@@ -216,12 +216,12 @@ theorem bodd_bit (b n) : bodd (bit b n) = b := by
   cases b <;> cases bodd n <;> simp [(show bodd 2 = false by rfl)]
 
 @[simp]
-theorem testBit_bit_zero (b) : ∀ n, testBit (bit b n) 0 = b
-  | (n : ℕ) => by rw [bit_coe_nat]; apply Nat.testBit_bit_zero
+theorem bit_testBit_zero (b) : ∀ n, testBit (bit b n) 0 = b
```

Comment:
This rename isn't consistent with `testBit_bit_succ` below

Reply:
Matching core seems sensible, but perhaps you should do the renames separately to the refactor for visibility.